app/routes.test.js

```diff
@@ -216,17 +216,48 @@ describe('routes', () => {
   });
 
   describe('POST /solutions/capabilities-selector', () => {
-    it('should return the correct status and text if there is no error', () => {
+    it('should return the correct status and text if there is no error and one capability selected', () => {
+      solutionListPageContext.getSolutionsForSelectedCapabilities = jest.fn()
+        .mockImplementation(() => Promise.resolve(mockFilteredSolutions));
+      const app = new App().createApp();
+      app.use('/', routes);
+      return request(app)
+        .post('/solutions/capabilities-selector')
+        .send({ capabilities: 'C1' })
+        .expect(302)
+        .then((res) => {
+          expect(res.redirect).toEqual(true);
+          expect(res.text).toEqual('Found. Redirecting to /solutions/capabilities-selector.C1');
+        });
+    });
+
+    it('should return the correct status and text if there is no error and many capabilities selected', () => {
+      solutionListPageContext.getSolutionsForSelectedCapabilities = jest.fn()
+        .mockImplementation(() => Promise.resolve(mockFilteredSolutions));
+      const app = new App().createApp();
+      app.use('/', routes);
+      return request(app)
+        .post('/solutions/capabilities-selector')
+        .send({ capabilities: 'C1 C2 C3' })
```

Comment:
Should this be passed in as ['C1', 'C2', 'C3']?

Reply:
not to our post endpoint - it comes in this format from the form submit

Reply:
don't get it. How else does this come to our post endpoint if not the form submit?

Reply:
it comes in this format from the form submit

Reply:
which bit is confusing you?